### ISSUE TITLE
sa_regions is scrambled!

### ISSUE THREAD

**Issue by hjundicodes** [usefksa/Saudi_GIS_Data#7]
Hi there! 

I appreciate the work and effort a lot, But please upload sa_regions.sql again.
Every time I clone it, Seems its scrambled.

Thanks.

**Comment by whamalek** [comment]
Yes, same problem here, Unicode Replace character is being used.
The file is garbage.

**Comment by hjundicodes** [comment]
Unfortunately its the most import file of them all 😢
